README.md

```diff
@@ -231,9 +231,32 @@ To validate an `.parquet` file, specify `parquetFile` and the path to the file,
   checks:
 ```
 
+#### Custom reader format
```

Comment:
I think we should just describe this as access to the core `spark.read` fluent API. With 1:1 fields in YAML, then link directly to data source documentation:
https://spark.apache.org/docs/2.3.1/sql-programming-guide.html#data-sources